src/access.d

```diff
@@ -410,3 +410,26 @@ extern (C++) bool checkAccess(Loc loc, Scope* sc, Expression e, Declaration d)
     }
     return false;
 }
+
+/****************************************
+ * Check access to package/module p for scope sc.
+ *
+ * Returns: true if the package is not accessible.
+ *
+ * Because we use a global symbol table tree for imported packages/modules,
```

Comment:
'We' (and 'you', 'I', etc.) should not be used in documentation. The sentence can be redone as:
```
Because a global symbol table tree is used for ...
```